crates/proto/src/tests/udp.rs

```diff
@@ -13,8 +15,7 @@ use crate::rr::{Name, RData, Record, RecordType};
 use crate::runtime::RuntimeProvider;
 use crate::udp::{UdpClientStream, UdpStream};
 use crate::xfer::dns_handle::DnsStreamHandle;
-use crate::xfer::{DnsRequest, DnsRequestSender};
-use crate::xfer::{DnsRequestOptions, FirstAnswer, SerialMessage};
+use crate::xfer::{DnsRequest, DnsRequestOptions, DnsRequestSender, FirstAnswer, SerialMessage};
```

Comment:
I like this change, but would prefer it to be left out of this already large commit.

Reply:
I think it's a leftover from the `std` feature flag, will change back for this PR